File: C/maior_posicao.c
#include <stdio.h>

int main()
{
    int n, posMaiorVl;
    double maiorVl;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    double vet[n];

    for (int i = 0; i < n; i++)
    {
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }

    maiorVl = vet[0];
    posMaiorVl = 0;
    for (int i = 0; i < n; i++)
    {
        if (vet[i] > maiorVl)
        {
            maiorVl = vet[i];
            posMaiorVl = i;
        }
    }

    printf("\nMAIOR VALOR = %.1lf\n", maiorVl);
    printf("POSICAO DO MAIOR VALOR = %d\n", posMaiorVl);
    return 0;
}

File: Python/maior_posicao.py
n = int(input("Quanto numeros voce vai digitar? "))
pos: int
maior: float

vet = [0 for x in range(n)]

for i in range(0, n):
    vet[i] = float(input("Digite um numero: "))

maior = 0;
pos = 0
for i in range(0, n):
    if vet[i] > maior:
        maior = vet[i]
        pos = i

print(f"\nMAIOR VALOR = {maior:.1f}")
print(f"POSICAO DO MAIOR VALOR = {pos}")